docker/lib/dependabot/docker/tag.rb

```diff
@@ -100,25 +117,28 @@ def format
         # That means only "22-ea-7" will be considered as a viable update
         # candidate for "21-ea-32", since it's the only one that respects that
         # format.
-        if version.match?(WORDS_WITH_BUILD)
-          return :"<version>#{version.match(WORDS_WITH_BUILD).to_s.gsub(/-[0-9]+/, '-<build_num>')}"
+        if T.must(version).match?(WORDS_WITH_BUILD)
+          return :"<version>#{T.must(version).match(WORDS_WITH_BUILD).to_s.gsub(/-[0-9]+/, '-<build_num>')}"
```

Comment:
IMO this `T.must` is okay, as we already verified on the line above that it's not `nil`.